src/test/kotlin/calculator/BinaryOperationBddTest.kt

```diff
@@ -0,0 +1,25 @@
+package calculator
+
+import calculator.common.model.PositiveInteger
+import calculator.domain.operation.AdditionBinaryOperation
+import calculator.domain.operation.BinaryOperationCommand
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class BinaryOperationBddTest : BehaviorSpec({
+
+    given("2개의 숫자 값이 주어졌을 때") {
+        val firstNumber = 1
+        val secondNumber = 3
+
+        `when`("이항식 더하기 연산을 하면") {
+            val binaryOperation = AdditionBinaryOperation
+            val binaryOperationCommand = BinaryOperationCommand(leftArgument = PositiveInteger(firstNumber), rightArgument = PositiveInteger(secondNumber))
+            val result = binaryOperation.operate(binaryOperationCommand)
+
+            then("연산이 된다") {
+                result shouldBe firstNumber + secondNumber
```

Comment:
결과값을 검증하는 과정에서 `binaryOperation` 의 기능(+)이 노출되었네요 🤔 
현재는 연산이 단순해서 티가 덜 나지만 내부 알고리즘이 테스트 코드에 노출되는 것은 안티 패턴인데요~

결과값은 하드코딩된 값으로 하는 것이 더 좋아보입니다!
```suggestion
                result shouldBe 4
```

Reply:
그렇군요. 노출을 안시키도로 의식적으로 코드를 짜야겠습니다 😢[ff0de46](https://github.com/next-step/kotlin-lotto/pull/462/commits/ff0de462bbc4cbddee9bc1780c22db4382eb3a08)